tests/unit/recommenders/utils/test_gpu_utils.py

```diff
@@ -38,12 +37,12 @@ def test_clear_memory_all_gpus():
 @pytest.mark.gpu
 @pytest.mark.skipif(sys.platform == "win32", reason="Not implemented on Windows")
 def test_get_cuda_version():
-    assert get_cuda_version() > "9.0.0"
+    assert int(get_cuda_version().split(".")[0]) > 9
```

Comment:
Shouldn't you handle the CUDA failure cases here, now that the functions return None?

Reply:
yeah, you're right. The functions to get cuda and to get cudnn should be consistent and either return None or catch an exception. I will make a new issue: https://github.com/recommenders-team/recommenders/issues/1994